src/Templates/Microsoft.NetCore.FSharp.ProjectTemplates.Test/MSTest.vstemplate

```diff
@@ -3,7 +3,7 @@
   <TemplateData>
     <Name Package="{52CBD135-1F97-2580-011F-C7CD052E44DE}" ID="521"/>
     <Description Package="{52CBD135-1F97-2580-011F-C7CD052E44DE}" ID="522"/>
-    <Icon Package="{52CBD135-1F97-2580-011F-C7CD052E44DE}" ID="531"/>
+    <Icon Package="{91A04A73-4F2C-4E7C-AD38-C1A68E7DA05C}" ID="4018" />
```

Comment:
Do we want the class library icon for F# test projects?  Is there no test icon?

Reply:
4018 is the newly added test icon.
4002 is the class library icon.